### ISSUE TITLE
example should not ignore err

### ISSUE THREAD

**Issue by jftuga** [otiai10/gosseract#156]
In your `README.md` example, you have:

    text, _ := client.Text()

I think it would be better to have:

    text, err := client.Text()
    if err != nil {
        fmt.Println(err)
    }

Once I did this, I actually saw a useful error message:

    PixImage is not set, use SetImage or SetImageFromBytes before Text or HOCRText

FWIW, this error was caused because I used the wrong path `client.SetImage()`, thus creating a *file not found* issue.  This is a rather odd error message for this circumstance.

**Comment by otiai10** [comment]
@jftuga Thank you! Can you summarize what the problem is and what it should be?

**Comment by jftuga** [comment]
I have listed 2 problems here - sorry about that.

Really the only thing I am suggesting is to change the Example in your `README.md` file to include error checking instead of ignoring it by using the `_` place holder.

**Comment by otiai10** [comment]
IMO, it's too sweet. Users can add the error check by theirselves
